modules/hal_nordic/nrfx/Kconfig

```diff
@@ -9,6 +9,10 @@ menu "nrfx drivers"
 
 rsource "Kconfig.logging"
 
+config MPSL
+	bool
+	# Dummy kconfig to allow compiling outside of NCS
```

Comment:
CI in https://github.com/nrfconnect/sdk-nrf/pull/15940 fails on building Zigbee samples that use overlays with `CONFIG_MPSL=n`.
I think that instead of the above you should introduce a Kconfig option like `MPSL_IN_USE` that will be selected by `MPSL`.

Reply:
There shouldn't be a need for `CONFIG_MPSL=n` in APP-core overlays anyways. Do you know the reason?

Reply:
I don't know. Perhaps the one who added such overlay could say something more.

Reply:
The CI is green. Could we get this in?
That fixes the immediate problem (samples not working at all).

We can make a nicer, generic solution after the release, so we can discuss it properly without the release stress.

Reply:
CI fails in manifest update (https://github.com/nrfconnect/sdk-nrf/pull/15940) because of build failures, so this PR cannot be merged.